### ISSUE TITLE
Subcutaneous abscess

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10687]
**Preferred term label:**
Subcutaneous abscess

**Synonyms**

**Definition (free text, please give PubMed ID)**
an abscess in the deeper layer of tissue below the skin

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0025615
Sibling term https://hpo.jax.org/browse/term/HP:0031292

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

 Pyogenic flexor tenosynovitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
see this for definitions, https://link.springer.com/chapter/10.1007/978-3-031-22599-4_114

**Comment by MickeySegal** [comment]
If you need more than "an abscess in the deeper layer of tissue below the skin" feel free to add it.

**Comment by pnrobinson** [comment]
This will need a little revision.
Derive defitions from https://link.springer.com/chapter/10.1007/978-3-031-22599-4_114

Cutaneous and subcutaneous abscesses are associated with characteristic clinical manifestations in the involved skin area. The patient suffers from the pain. Traditional local manifestations of abscesses include warmth, redness, and swelling. The characteristic morphologic feature of the cutaneous/subcutaneous abscess is fluctuant lesion. It means palpable motion caused by movement of the pus inside the abscess. Abscess may manifest without cellulitis, or it can be associated with surrounding cellulitis. In case of advanced cellulitis, identifying the exact location of the abscess may be difficult. Sometimes there are swollen lymph nodes—regional lymphadenopathy. Symptoms of systemic toxicity (fever, shivering fit, fatigue) are unusual. Late clinical manifestations include abscess perforation. Abscesses eventually burst and spontaneously drain yellow pus.
 
 
Abscess and cellulitis are the most common types of pyogenic skin and soft tissue infections in surgery. Pyogenic skin and soft tissue infection is characterised by the production of purulent exudate. Cutaneous and subcutaneous abscesses are defined as a localised collection of pus within the dermis or subcutaneous space. Cellulitis is interstitial purulent inflammation spreading diffusely in the dermis and subcutaneous tissue. Both forms of exudative purulent inflammation are caused by pyogenic (pus-producing) bacteria.
 
Abscess is caused by bacteria that enter and infect the cutaneous and subcutaneous tissue through breaks in the skin, such as minor skin trauma, burns and insect bites, or hair follicles. Therefore, the microbial aetiology of abscesses often comes from indigenous bacteria flora colonising the skin of healthy individuals (Gram-positive cocci). However, skin abscesses can be caused by a wide spectrum of exogenous bacteria infiltrating the skin and soft tissue during the trauma as well. Cutaneous and subcutaneous abscesses are usually monomicrobial, but they can be caused by more than one pathogen.
 
Standard classification of cutaneous and subcutaneous abscesses distinguishes simple superficial abscesses from complex abscesses. Superficial simple abscesses are usually well-circumscribed and uniloculated. These abscesses should not have an extension into deeper tissues. Complex abscesses are characterised by their specific location (perineal, perianal, axillary), the pathogens involved (polymicrobial, resistant pathogens), and patient characteristics (e.g., immunodeficiency). advantageous or an obligatory contracture.

**Comment by MickeySegal** [comment]
This is a lot of detail; I was thinking of something along the lines of https://uts.nlm.nih.gov/uts/umls/concept/C1385517.

There is also https://uts.nlm.nih.gov/uts/umls/concept/C0241266 but that is considered a disease, but it is also a finding.

**Comment by pnrobinson** [comment]
@MickeySegal 


I noticed we already have this:
[Subcutaneous abscess HP:6001359](https://hpo.jax.org/browse/term/HP:6001359)
But the link to Abscess was missing, I have added it.
